test/e2e/specs/site-editor/pages.spec.js

```diff
@@ -78,7 +78,7 @@ test.describe( 'Pages', () => {
 		).toBeVisible();
 
 		// Switch to template editing focus.
-		await page.getByRole( 'button', { name: 'Settings' } ).click();
+		await editor.openDocumentSettingsSidebar();
```

Comment:
It's unrelated to the fix, but it's better to use the util to avoid accidentally closing the 'Settings' sidebar if it is open.